style.css

```diff
@@ -342,3 +349,101 @@ nav .status-bar {
   font-size: 22px;
   line-height: 43px;
 }
+
+.contact-section {
+  background: white;
+}
+
+.contact h3 {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  padding-top: 120px;
+}
+
+.contact-list {
+  border-bottom: 2px solid #dbd8d7;
+  padding-bottom: 20px;
+}
+
+input {
+  outline: none;
+  font-size: 15px;
+  line-height: 24px;
+  padding-top: 70px;
+  font-style: normal;
+  font-weight: normal;
+  color: #979493;
+  padding-bottom: 30px;
+}
+
+textarea {
+  background: #fbf8f7;
+  font-size: 15px;
+  line-height: 24px;
+  margin-right: 24px;
+  color: #3c3a39;
+  font-style: normal;
+  font-weight: normal;
+  padding-top: 20px;
+  margin-bottom: 30px;
+}
+
+textarea::placeholder {
+  color: #3c3a39;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 160%;
+}
+
+.btn {
+  background-color: #ff6b00;
+  text-align: center;
+  width: 50%;
+  align-items: center;
+  padding: 12px;
+  display: block;
+  height: 48px;
+  margin: auto;
+}
+
+.btn:hover {
+  background-color: #091e42;
+}
```

Comment:
- [ ] While you have made a great effort to add button interactions, it would be better to stick to the design. Kindly fix that.
![image](https://user-images.githubusercontent.com/59416136/154057170-c610fe9f-60e5-4042-ae52-c3f5bf5adae7.png)